### ISSUE TITLE
calcMonAnom

### ISSUE THREAD

**Issue by clyne** [NCAR/geocat-comp#13]
Port the **calcMonAnom** family of functions, including: calcMonAnomTLL, calcMonAnomTLLL. The **calcMonAnom** functions calculates monthly anomalies by subtracting the long term mean from each point (time,lat,lon version)

**Comment by andersy005** [comment]
Just wanted to point out this has been implemented in #100


```python
In [1]: import geocat.comp
import 
In [2]: import xarray as xr

In [3]: ds = xr.tutorial.open_dataset('rasm')

In [5]: geocat.comp.anomaly(ds, 'month')
Out[5]: 
<xarray.Dataset>
Dimensions:  (time: 36, x: 275, y: 205)
Coordinates:
    xc       (y, x) float64 ...
    yc       (y, x) float64 ...
  * time     (time) object 1980-09-16 12:00:00 ... 1983-08-17 00:00:00
    month    (time) int64 9 10 11 12 1 2 3 4 5 6 7 ... 10 11 12 1 2 3 4 5 6 7 8
Dimensions without coordinates: x, y
Data variables:
    Tair     (time, y, x) float64 nan nan nan nan ... 0.4289 0.4815 0.4457
```

**Comment by anissa111** [comment]
> Just wanted to point out this has been implemented in #100

Oh yup I can see that, thank you for pointing that out!
